Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -41,17 +35,17 @@ ILogger<PostsController> logger
         [Authorize]
         [HttpPut("{id}")]
         public async Task<IActionResult> PutAsync(int id, Post post) {
-            /*try {
+            try {
                 var sessionEmail = User.FindFirst(ct => ct.Type.Contains("nameidentifier")).Value;
-                var postEmail = _postsRepository.First(p => p.Id == id).UserEmail;
+                var postEmail = _postsRepository.AsQueryable().First(p => p.Id == id).UserEmail;
```

Comment:
i'm not sure calling AsQueryable here does what you want it to do or not.

Reply:
To use linq method or their async versions in EntityFrameworkCore we need to call this on the repo, it doesn't pull any data just lets us call the extension methods like the AsEnumerable method.